images/base-images/nodejs/image-template/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG SYSTEM_PACKAGES_FILE=system-packages.txt
+COPY ${SYSTEM_PACKAGES_FILE} .
+
+RUN packages=$(cat ${SYSTEM_PACKAGES_FILE} | sed 's/ /-/' | uniq | paste -d' ' -); \
```

Comment:
Are you changing the format of the packages file?  It was a structured json document, but it looks like you're moving to a txt list.  I think we should keep the structure.  Also, I don't know why you are substituting?